spec/models/links_udpate_spec.rb

```diff
@@ -69,12 +67,24 @@
     end
 
     it 'adds an error message' do
+      stub_request(:post, "https://publishing-api.test.gov.uk/lookup-by-base-path")
+        .with(body: "{\"base_paths\":[\"/a/base/path\"]}")
+        .to_return(status: 200, body: "{}")
+
+      tag_mapping = create(:tag_mapping, link_type: 'non-taxons')
+      links_update = build(:links_update, links: { 'non-taxons' => ['a-taxon-content-id'] })
+      links_update.tag_mappings = TagMapping.all
+
       expectation = lambda do
```

Comment:
@carvil I've changed it to the following instead:

```
      expect(tag_mapping.message).to_not match(/Content We could not find/)
      expect(tag_mapping.message).to_not match(/Link types/)
      expect(tag_mapping.message).to match(/invalid link types found/i)
```

Since we were aiming to not have the type in the beginning of each error message (in this case is `Content` and `Link types`)